pkg/load/proxy.go

```diff
@@ -61,12 +61,18 @@ func RunProxyImage(ctx context.Context, dockerapi docker.APIClient, proxyImage s
 		return nil, err
 	}
 
+	attach, err := dockerapi.ContainerAttach(ctx, resp.ID, types.ContainerAttachOptions{Stdin: true, Stdout: true, Stderr: true, Logs: true, Stream: true})
```

Comment:
I found out that container attach needs to be before container start.  Despite the docs saying otherwise, it turns out that logging messages (such as our alive check) could be dropped unless you start attach first.